ktoml-core/src/commonMain/kotlin/com/akuleshov7/ktoml/tree/TomlInlineTable.kt

```diff
@@ -75,4 +76,31 @@ public class TomlInlineTable(
         }
         return tomlTable
     }
+
+    public override fun write(
+        emitter: TomlEmitter,
+        config: TomlConfig,
+        multiline: Boolean
+    ) {
+        val key = TomlKey(name, 0)
+
+        key.write(emitter, config)
+
+        emitter.emitPairDelimiter()
+
+        emitter.startInlineTable()
+
+        tomlKeyValues.forEachIndexed { i, pair ->
+            if (i > 0) {
+                emitter.emitElementDelimiter()
+            }
+
+            emitter.emitWhitespace()
+
+            pair.write(emitter, config)
+        }
+
+        emitter.emitWhitespace()
```

Comment:
I have a small suggestion here - may be we will change emmiter API in the following way:
- all methods will return `this` in a functional way
- so we would be able to use it in the functional way:
```
        emitter.emitWhitespace()
                    .endInlineTable()
                    .enitNewLine()
... 
```